modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-vm-suma.adoc

```diff
@@ -66,13 +67,15 @@ If you want to use a production environment and need background information abou
 | Installation Method | Import Existing Disk Image
 | OS:                 | Linux
 | Version:            | {productname}-Proxy.x86_64-5.0.0-*.qcow2
-| Memory:             | 2 GB (minimum)
-| CPU's:              | 2
-| Storage Format:     | _.qcow2_ 40 GB (Default) Root Partition
+| Memory:             | Minimum *)
+| CPU's:              | Minimum *)
+| Storage Format:     | _.qcow2_ 20 GB (Default) Root Partition
```

Comment:
I discussed this with @rjmateus and we agreed to increase VM images to 40GB. Particularly that we have enough space for different MU container images.

Even though size of VM is not directly related to the minimum storage requirements, I think we should rather change to 40GB minimum everywhere.